Ginger/GingerCoreNET/GeneralLib/EmailReadMSGraphOperations.cs

```diff
@@ -486,6 +556,19 @@ private void AppendFromFilter(StringBuilder filterParameter, EmailReadFilters fi
 
             filterParameter.Append($"from/emailAddress/address eq '{filters.From}'");
         }
+        private void AppendReadUnreadFilter(StringBuilder filterParameter, EmailReadFilters filters)
+        {
+
+            if (filters.ReadUnread.Equals(true))
```

Comment:
Please remoe .Equals if ReadUnread is boolean as if will pass if ReadUnread is set to true